content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Configuring Azure AD as the SAML IDP with Greenlake Cloud Platform and Aruba Central
+date: 2022-05-09T12:04:30.750Z
+author: Michael Rose Jr.
+authorimage: /img/mrose-photoprofile.png
+tags:
+  - aruba
+  - aruba-central
+---
+
+Single sign-on (SSO) enables users to securely authenticate with multiple applications and websites by logging in only once—with just one set of credentials (username and password). With SSO, the application or website that the user is trying to access relies on a trusted third party (Identity provider) to verify that users are who they say they are.
+
+I'll explain the process for configuring Azure AD to authenticate users into Greenlake Cloud Platform (GLCP) and Aruba Central using SAML IDP.
+
+If you're looking for the Okta version of this guide can be found on [WIFI-GUYS](https://www.wifi-guys.com/?p=512)
+<!-- prettier-ignore-start -->
+## Contents
```

Comment:
I left the Content section in as the blog is long enough to warrant it. I can used as a reference doc and easier to find content rather than having to scroll up and down constantly. I've always found that annoying on longer blog posts.